### ISSUE TITLE
Clink already loaded in process XXXX

### ISSUE THREAD

**Issue by bw1faeh0** [chrisant996/clink#114]
Starting with one one of the newer updates clink tells me everytime I start my console, that Clink is already loaded in process XXXX.
![image](https://user-images.githubusercontent.com/29235332/118438310-6d733e80-b6e4-11eb-8b08-354324e71b8c.png)

It does not bother me within my console/terminal, but when other proccesses use a console for logging, it is resulting in a lot of these messages in my log files (i.e. console output of Eclipse compiling and linking some code).

![image](https://user-images.githubusercontent.com/29235332/118438734-1ae65200-b6e5-11eb-90bc-5b5dbf84a0a5.png)

This is raising always questions when reviewing logs with colleagues....

Can we disable this message?

**Comment by chrisant996** [comment]
Why is something trying to inject Clink multiple times into the same process?

The second one doesn't work. So options in the second one don't take effect, and the outcome is not what was expected. An error message is appropriate.

Can the problem of multiple injection attempts be corrected?

What does `clink autorun show` report?

**Comment by bw1faeh0** [comment]
````
C:\Users\bw1faeh0>clink autorun show
Current AutoRun values

  Current user:

    native : "D:\Programme\clink\1.2.6.4172bb\clink.bat" inject --autorun
     wow64 : "D:\Programme\clink\1.2.6.4172bb\clink.bat" inject --autorun

  All users:

    native : "D:\Programme\clink\1.2.6.4172bb\clink.bat" inject --autorun
     wow64 : "D:\Programme\clink\1.2.6.4172bb\clink.bat" inject --autorun
````

**Comment by chrisant996** [comment]
Ah, that's the problem:  Somehow autorun has been set up for both All Users and also Current User.  So Clink gets injected twice.

One or the other should be set, but not both.

You could use `clink autorun uninstall` to clear the Current User autorun.
Or use `clink autorun uninstall -a` to clear the All Users autorun.

**Comment by chrisant996** [comment]
(But also, I'll see if I can do something to suppress the error when the multiple injection requests are happening during autorun.)

**Comment by bw1faeh0** [comment]
I think I installed autorun for my user first and later installed it for all users. I wasn't aware that you have to choose one OR the other option. 
Now, I cleared the autorun for my user.

**Comment by chrisant996** [comment]
> I think I installed autorun for my user first and later installed it for all users. I wasn't aware that you have to choose one OR the other option. 

It hadn't occurred to me that autorun could get set up with both. I'll see if I can do something to suppress the error when the multiple injection requests are happening during autorun.

**Comment by chrisant996** [comment]
Thanks for reporting this!  Should be fixed in v1.2.8 when it gets published later this week.
